syntax = "proto2";

message response_msg
{
	required string info = 1;
}